econml/tests/test_orf.py

```diff
@@ -49,7 +51,8 @@ def test_continuous_treatments(self):
             T * TE + TestOrthoForest.epsilon_sample(TestOrthoForest.n)
         # Instantiate model with most of the default parameters
         est = ContinuousTreatmentOrthoForest(n_jobs=4, n_trees=10,
-                                             model_T=Lasso(), model_Y=Lasso(),
+                                             model_T=WeightedModelWrapper(Lasso(), sample_type="weighted"),
```

Comment:
`model_T` and `model_Y` don't need to accept weights because the nuisance estimates in the causal tree are not weighted.